test/stream/api/sink/statistics-total-001-ok/002-poll-source/request.json

```diff
@@ -0,0 +1,11 @@
+{
+  "path": "<<001-create-source:response.url>>",
+  "method": "GET",
+  "headers": {
+    "X-StorageApi-Token": "%%TEST_KBC_STORAGE_API_TOKEN%%"
+  },
+  "repeat": {
+    "until": "status != 'processing'",
+    "timeout": 60
+  }
```

Comment:
This repeats request on `stream API`  till timeout of status is not `processing` something like `testify.Eventually()`?